src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -328,19 +330,81 @@ private void InsertEntry(NodeSearchElementViewModel entry, IEnumerable<string> c
             var nameStack = new Stack<string>(categoryNames.Reverse());
```

Comment:
A bit cahnged as this method works not only for zero-touch function but for all types of elements (node model function, custom node)
```C#
/// <summary>
        /// Insert a new search element under the category.
        /// </summary>
        /// <param name="entry">This could represent a function of a given 
        /// class. For example, 'MyAssembly.MyNamespace.MyClass.Foo'.</param>
        /// <param name="categoryNames">A list of entries that make up the fully qualified
        /// class name that contains function 'Foo', e.g. 'MyAssembly.MyNamespace.MyClass'.
        /// </param>
```